<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTimelineWidget</class>
 <widget class="QWidget" name="MainTimelineWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1138</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>41</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget #widget {
border-bottom: 1px solid #383838;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QWidget" name="controlsWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>400</width>
          <height>40</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="start">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#start {
background-color: rgba(120, 120, 120,0);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}
QPushButton#start:hover {
background-color: rgb(30, 144, 255);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/app/icons/start.svg</normaloff>:/app/icons/start.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fastRewind">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#fastRewind {
background-color: rgba(120, 120, 120,0);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}
QPushButton#fastRewind:hover {
background-color: rgb(30, 144, 255);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/app/icons/fastreverse.svg</normaloff>:/app/icons/fastreverse.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rewind">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#rewind {
background-color: rgba(120, 120, 120,0);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}
QPushButton#rewind:hover {
background-color: rgb(30, 144, 255);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/app/icons/reverse.svg</normaloff>:/app/icons/reverse.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stop">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#stop {
background-color: rgba(120, 120, 120,0);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}
QPushButton#stop:hover {
background-color: rgb(30, 144, 255);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/app/icons/stop.svg</normaloff>:/app/icons/stop.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="play">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#play {
background-color: rgba(120, 120, 120,0);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}
QPushButton#play:hover {
background-color: rgb(30, 144, 255);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/app/icons/play.svg</normaloff>:/app/icons/play.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fastForward">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#fastForward {
background-color: rgba(120, 120, 120,0);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}
QPushButton#fastForward:hover {
background-color: rgb(30, 144, 255);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/app/icons/fastforward.svg</normaloff>:/app/icons/fastforward.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>12</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="end">
           <property name="minimumSize">
            <size>
             <width>54</width>
             <height>40</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton#end {
background-color: rgba(120, 120, 120,0);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}
QPushButton#end:hover {
background-color: rgb(30, 144, 255);
border-style: solid;
border-color: rgb(138, 138, 138);
max-width:50px;
min-width:50px;
padding: 2px 2px 2px 2px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../icons.qrc">
             <normaloff>:/app/icons/end.svg</normaloff>:/app/icons/end.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>12</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>1</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget #widget {
border-bottom: 1px solid #383838;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTimeEdit" name="timeStart">
        <property name="displayFormat">
         <string>h:mm </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="TimelineWidget" name="timeline" native="true">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTimeEdit" name="timeEnd">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="displayFormat">
         <string>h:mm </string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimelineWidget</class>
   <extends>QWidget</extends>
   <header>src/widgets/timelinewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
